### DISCUSSION TITLE
Start ollama-webui at boot

### DISCUSSION THREAD

**Discussion by Gualty** [open-webui/open-webui#666] [Q&A] [answered]
Hi I followed the steps to install ollama-webui without Docker on my Raspberry Pi 5 with Raspbian.
I tried to edit cron job to start on reboot but it will not start. It will only start if I execute it from the normal user with ./start.sh until I close the terminal window.

Any suggestions?

**Comment by justinh-rahb** [comment, marked as answer]
It's legitimately probably easier to install Docker than it would be to explain how to do this the way you're going. This project is very optimized for Docker deployment, which is why that's the only supported method currently. What you're asking is indeed possible but for many reasons is not recommended.

**Reply by Gualty** [reply]
Thank you for your answer. I preferred not to install on docker because it is running on a raspberry and i want to preserve the resources for only ollama and ollama-webui.

**Reply by justinh-rahb** [reply]
> Thank you for your answer. I preferred not to install on docker because it is running on a raspberry and i want to preserve the resources for only ollama and ollama-webui.

Docker itself doesn't use much, I think you're probably overestimating the amount of overhead you'd experience.

**Reply by Gualty** [reply]
Ok, then it will be possibile to start the ollama-webui at startup?

**Reply by justinh-rahb** [reply]
Yes, make sure you set `--restart unless-stopped` or `--restart always` in your Docker run command, or it should already be specified in the Compose files if you're using that.

**Reply by Gualty** [reply]
Thank you!

**Comment by cap-steve** [comment]
i'm running a VM with GPU passthru and didnt feel like going down the docker route.
i know this is coming in late for @Gaulty, but i solved this as a service like this:

/etc/systemd/system/openwebui.service 
[Unit]
Description=OpenWebUI Service
After=network-online.target

[Service]
ExecStart=bash /home/<your username>/open-webui/backend/start.sh
User=<your username>
Group=<your username>
Restart=always
RestartSec=60
Environment="PATH=/home/<your username>/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"

[Install]
WantedBy=default.target
